src/lib/expansion/expansion.md

```diff
@@ -0,0 +1,81 @@
+`<md-expansion-panel>` provides a way to show and hide lightweight content, by collapsing and expanding a view with a nice animation.
+
+<!-- example(expansion-overview) -->
+
+The `expanded` input allows to choose among the collapsed or expanded state. 
+
+Each expansion panel has a header and an action sections. The header section is always visible at the top of the component and contains a title and a description subsections. The action section is fixed at the bottom, being visible when the expansion is in expanded state.
+
+When grouped by an `<md-accordion>` element the expansion panels can be used to create ordered views or flows, as it brings up the possibility to expand one view at a time.
```

Comment:
I would eliminate this sentence since we talk about accordions in their own section.